modules/ROOT/pages/db/db-connector-index.adoc

```diff
@@ -4,15 +4,18 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../../assets/images/
 
```

Comment:
Apparently, you can't have blank lines when defining metadata. 
I changed this:
```
:imagesdir: ../../assets/images/

:page-aliases: db/db-configure-connection.adoc
```
To:
```
:imagesdir: ../../assets/images/
:page-aliases: db/db-configure-connection.adoc
```